apps/lend/src/components/PageLoanManage/Page.tsx

```diff
@@ -132,6 +154,26 @@ const Page: NextPage = () => {
   return (
     <>
       <DocumentHead title={`${collateral_token?.symbol ?? ''}, ${borrowed_token?.symbol ?? ''} | Manage Loan`} />
+
+      {chartExpanded && networks[rChainId].pricesData && (
+        <PriceAndTradesExpandedContainer>
+          <Box flex padding="0 0 var(--spacing-2)">
+            <ExpandButton
+              variant={'select'}
+              onClick={() => {
```

Comment:
Why don't use just the function itself?
```suggestion
              onClick={setChartExpanded}
```